internal/persistence/ent/schema/user.go

```diff
@@ -44,12 +44,12 @@ func (User) Fields() []ent.Field {
 			Optional(),
 		field.String("oidc_iss").
 			Annotations(entsql.Annotation{
-				Size: 1024,
+				Size: 255,
```

Comment:
I'm a bit worried there could be edge cases where the issuer URL is longer than 255 chars. 🤔